planning/autoware_planning_validator/src/utils.cpp

```diff
@@ -78,11 +78,11 @@ Trajectory resampleTrajectory(const Trajectory & trajectory, const double min_in
 
 // calculate curvature from three points with curvature_distance
 void calcCurvature(
-  const Trajectory & trajectory, std::vector<double> & curvature_arr,
+  const Trajectory & trajectory, std::vector<double> & curvature_vector,
   const double curvature_distance)
 {
   if (trajectory.points.size() < 3) {
-    curvature_arr = std::vector<double>(trajectory.points.size(), 0.0);
+    curvature_vector = std::vector<double>(trajectory.points.size(), 0.0);
```

Comment:
we can move outside of {}

Reply:
Thanks! addressed in https://github.com/autowarefoundation/autoware_universe/pull/10385/commits/69a27ce5729676890081aa34711f3a6f515c3f9a